netplan_cli/cli/state.py

```diff
@@ -491,6 +504,46 @@ def query_resolved(cls) -> tuple:
             logging.debug('Cannot query resolved DNS data: {}'.format(str(e)))
         return (addresses, search)
 
+    @classmethod
+    def query_members(cls, ifname: str) -> List[str]:
+        members = []
+        output: str = None
+        try:
+            output = subprocess.check_output(
+                ['ip', '-d', '-j', 'link', 'show', 'master', ifname], text=True)   # wokeignore:rule=master
+        except Exception as e:
+            logging.warning('Cannot query bridge: {}'.format(str(e)))
+            return []
+
+        output_json = json.loads(output)
+        for member in output_json:
+            members.append(member.get('ifname'))
+
+        return members
+
+    @classmethod
+    def correlate_members_and_uplink(cls, interfaces: List[Interface]) -> None:
```

Comment:
nitpick (non-blocking): A brief docstring describing what this method does would be nice.